src/GitVersion.Core.Tests/Core/GitVersionExecutorTests.cs

```diff
@@ -22,7 +22,7 @@ public class GitVersionExecutorTests : TestBase
     private IGitVersionCache gitVersionCache;
     private IServiceProvider sp;
 
-    [Test]
+    [Test, Ignore("")]
```

Comment:
Please write a comment why this test is ignored.

Reply:
It was an accident. Sorry. I have changed it back.